### PR TITLE
MAUI-4750 Overview UG for Shimmer in .NET MAUI

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- I have added overview UG for shimmer in .NET MAUI

### CODE REVIEW

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/Shimmer/overview.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Shimmer (SfShimmer) control, its basic features, elements and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Overview of .NET MAUI Shimmer (SfShimmer)
+
+Syncfusion .NET MAUI Shimmer (SfShimmer) control can be used to increase an application's responsiveness by displaying a shimmer effect while the content is loading or processing in the background.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/overview.md]: Comments addressed

File: MAUI/Shimmer/overview.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Key features
+
+* **Multiple Shimmer View types**: There are seven different Shimmer view types are available in the .NET MAUI `SfShimmer`. `CirclePersona` view is initially rendered by default.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/overview.md]: Comments addressed

File: MAUI/Shimmer/overview.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+* **Multiple Shimmer Wave Directions**: It defines the Animation wave direction for the Shimmer. There are five wave directions are available in `SfShimmer`. By default, wave direction is rendered by `Default` wave, it allows the animation from left top to right bottom.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/overview.md]: Comments addressed

File: MAUI/Shimmer/overview.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+* **Shimmer Custom view**: `SfShimmerView` supports `ShimmerView`, can be used in Custom view to change the Shimmer shape types.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/overview.md]: Comments addressed

File: MAUI/Shimmer/overview.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Syncfusion .NET MAUI Shimmer (SfShimmer) control can increase an application's responsiveness by displaying a shimmer effect while the content is loading or processing in the background.
+
+## Key features
+
+* **Multiple Shimmer View types**: There are seven different Shimmer view types available in the .NET MAUI `SfShimmer`. The `CirclePersona` view is initially rendered by default.
+
+* **Multiple Shimmer Wave Directions**: It defines the Animation wave direction for the Shimmer. There are five wave directions available in the `SfShimmer`. By default, the wave direction is rendered by the `Default` wave, allowing the animation from the left top to the right bottom.
+
+* **Shimmer Custom view**: `SfShimmerView` supports `ShimmerView`, which can be used in Custom view to change the Shimmer shape types.
```
Comment (Sasikumar3595) [inline MAUI/Shimmer/overview.md:20]: `SfShimmerView` supports `ShimmerView` -- Check the meaning

Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/overview.md]: Checked and shimmer view content modified

**Pull request merged**